### DISCUSSION TITLE
WireUI from subfolder on server

### DISCUSSION THREAD

**Discussion by levinst** [wireui/wireui#241] [Q&A]
Tell me what could be the problem? The project is located in a subfolder on the server (`localhost/mintrans`), when I try to open the dialog, an error occurs in the console:

`GET http://localhost/wireui/icons/outline/question-mark-circle 404 not found`

loading components such as buttons, icons do not take into account the path to the subfolder as it should be:
`http://localhost/mintrans/wireui/icons/outline/question-mark-circle`
Livewire works fine.

**Comment by ph7jack** [comment]
Is your laravel app url with the subfolder path?

**Comment by insulae** [comment]
> 
did you find a solution to this?

**Reply by insulae** [reply]
I have a "ugly" fix: 

https://github.com/wireui/wireui/discussions/703
